README.md

```diff
@@ -51,6 +51,28 @@ defmodule MyApp.Repo.Migrations.CreateJobQueue do
 end
 ```
 
+### Upgrading pre-existent Ecto Job to 3.x.x version
+
+To upgrade your project to 3.x.x version of ecto job you must add a migration to update the pre-existent job queue tables:
+
+```
+mix ecto.gen.migration update_job_queue
+```
+
+```elixir
+defmodule MyApp.Repo.Migrations.UpdateJobQueue do
+  use Ecto.Migration
+
+  def up do
+    EctoJob.Migrations.AddPriorityToJobTable.up("jobs")
```

Comment:
Would it be easier for users to have a function to migrate the table up to a particular version?

```elixir
def up do
  EctoJob.Migrations.Update.up(3, "jobs")
end
```

Reply:
@mbuhot Done!